### PR TITLE
RDoc-770 setup-aggressive-caching

### PR DESCRIPTION
**Pull request by IdanHaim**

### CODE REVIEW

File: Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/HowTo/SetupAggressiveCaching.cs [hunk 1]
```diff
@@ -19,12 +20,15 @@ public SetupAggressiveCaching()
 			{
 				documentStore.Initialize();
 
-                #region max_number_of_requests
-			    // TODO change this to MaxHttpCacheSize 
-                documentStore.Conventions.MaxNumberOfRequestsPerSession = 1;
-				#endregion
+                #region max_number_of_requests 
+                documentStore.Conventions.MaxHttpCacheSize = new Size(1024, SizeUnit.Bytes);
```
Comment (ppekrol) [inline Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/HowTo/SetupAggressiveCaching.cs]: MegaBytes?

Comment (IdanHaim) [reply Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/HowTo/SetupAggressiveCaching.cs]: It is just an example that you can change the size to what sizeUnit you want even bytes

Comment (ayende) [reply Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/HowTo/SetupAggressiveCaching.cs]: But it looks very strange for the user

Comment (IdanHaim) [reply Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/HowTo/SetupAggressiveCaching.cs]: So you think I should change it to MegaBytes?

Comment (ayende) [reply Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/HowTo/SetupAggressiveCaching.cs]: yes

Comment (IdanHaim) [reply Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/HowTo/SetupAggressiveCaching.cs]: done

**Pull request merged**